core/src/main/groovy/noe/tomcat/configure/TomcatConfigurator.groovy

```diff
@@ -367,6 +367,46 @@ class TomcatConfigurator {
     return this
   }
 
+  /**
+   * Removes the specified listener to server.xml
+   * @param listenerFQCN the fully qualified class name of the listener
+   * @return this
+   */
+  TomcatConfigurator removeListener(String listenerFQCN) {
```

Comment:
can we have unit test for this method, please?

Reply:
I made an unit test about removeListener and addListener.

Reply:
Could you please share the code?